test/goldens/end2end/_dummy_pkg-1.0.0-null-safety.0.json

```diff
@@ -23,32 +23,7 @@
     "name": "BSD",
     "url": "https://github.com/dart-lang/pub-dev/blob/master/LICENSE"
   },
-  "pkgResolution": {
```

Comment:
Such a huge difference seems to be really off.

Not sure if this is the new version or a different SDK, but CI and end2end tests are on `2.12.0-0.0.dev`. Can you regenerate them with that SDK?